DfE.FindInformationAcademiesTrusts/Program.cs

```diff
@@ -109,7 +110,12 @@ private static HeaderPolicyCollection GetSecurityHeaderPolicies()
                 cspBuilder.AddScriptSrc()
                     .Self()
                     .UnsafeInline()
-                    .WithNonce();
+                    .WithNonce()
+                    .From("https://js.monitor.azure.com/scripts/b/ai.2.min.js");
+                cspBuilder.AddConnectSrc()
+                    .Self()
+                    .WithNonce()
```

Comment:
Do we need `WithNonce` on the connect src?

Reply:
I think I left that in from a test which didn't work. I'll remove it.